app/controllers/conference_attendances_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ConferenceAttendancesController < ApplicationController
+
+  def update
+    @conference_attendance = ConferenceAttendance.find(attendance_params[:id])
+    @team = Team.find(attendance_params[:team_id])
+
+    if @conference_attendance.update_attributes(attendance_params)
+      redirect_to @team, notice: 'Your attendance was successfully recorded.'
+    else
+      render json: @team.errors, status: :unprocessable_entity
+    end
+   
+  end
+
+  private
+    def attendance_params
+      params.permit(:attendance, :id, :team_id) 
```

Comment:
Permitting `id`  in the `attendance_params` suggests that it is part of the data that need to be changed. The way to handle finding a record in an update action would probably be to use `params[:id]`. Also the rest of the params allow the setting of the `:team_id`, is that really intended? I think the controller name suggests that one should only be able to change the attendance attribute. One last thought: the params should probably be nested under `conference_attendance`?